javaparser-core/src/main/java/com/github/javaparser/ast/visitor/SimpleVoidVisitor.java

```diff
@@ -0,0 +1,337 @@
+package com.github.javaparser.ast.visitor;
+
+import com.github.javaparser.ast.*;
+import com.github.javaparser.ast.body.*;
+import com.github.javaparser.ast.comments.BlockComment;
+import com.github.javaparser.ast.comments.JavadocComment;
+import com.github.javaparser.ast.comments.LineComment;
+import com.github.javaparser.ast.expr.*;
+import com.github.javaparser.ast.modules.*;
+import com.github.javaparser.ast.stmt.*;
+import com.github.javaparser.ast.type.*;
+
+public class SimpleVoidVisitor<A> implements VoidVisitor<A> {
```

Comment:
Is this visitor interesting for the core library? How would you describe what to use it for?

Reply:
Yes, the Checker Framework uses it once.
It is extended to another visitor class that overrides only needed methods for those case.
You could check the code here: https://github.com/typetools/checker-framework/blob/master/framework/src/org/checkerframework/framework/stub/StubUtil.java#L211-L333

Reply:
Made an issue for it here: https://github.com/javaparser/javaparser/issues/1140

Reply:
@matozoid thank you!